packages/hop-node/src/watchers/classes/Bridge.ts

```diff
@@ -645,6 +645,9 @@ export default class Bridge extends ContractBase {
     }
 
     const bonderFeeBps = fees[destinationChain]
+    if (!bonderFeeBps) {
```

Comment:
Is this validation needed? IIUC, this is validated in the watcher creation which occurs prior to this.

Reply:
I think it can be removed but it can be bad if for whatever reasons there is no bonder fee here so I like having the assertion check in there. I think the class shouldn't make assumptions

Reply:
Makes sense & SGTM.